### PR TITLE
Suprememods

### PR DESCRIPTION
**Pull request by supreme2580**

### CODE REVIEW

File: src/app/app.ts [hunk 1]
```diff
@@ -65,18 +65,18 @@ client.login(process.env.DISCORD_BOT_TOKEN).catch(Log.error);
...
-			Intents.FLAGS.GUILD_BANS,
-			Intents.FLAGS.GUILD_MEMBERS,
-			Intents.FLAGS.GUILD_EMOJIS_AND_STICKERS,
-			Intents.FLAGS.GUILD_VOICE_STATES,
-			Intents.FLAGS.GUILD_PRESENCES,
-			Intents.FLAGS.GUILD_MESSAGES,
-			Intents.FLAGS.GUILD_MESSAGE_REACTIONS,
-			Intents.FLAGS.DIRECT_MESSAGES,
-			Intents.FLAGS.DIRECT_MESSAGE_REACTIONS,
+			"Guilds",
```
Comment (tikisailor) [inline src/app/app.ts:68]: these should look like this `GatewayIntentBits.Guilds` I believe

File: src/app/events/RateLimit.ts [hunk 2]
```diff
@@ -8,13 +8,13 @@ export default class implements DiscordEvent {
...
-			Log.warn(`rate limit reached timeout: ${rateLimitData.timeout}`, {
+			Log.warn(`rate limit reached timeout: ${rateLimitData.timeToReset}`, {
 				indexMeta: true,
 				meta: {
-					timeout: rateLimitData.timeout,
+					timeout: rateLimitData.timeToReset,
 					limit: rateLimitData.limit,
 					method: rateLimitData.method,
-					path: rateLimitData.path,
+					path: rateLimitData.majorParameter,
```
Comment (tikisailor) [inline src/app/events/RateLimit.ts:17]: maybe `url: rateLimitData.url`? but not sure if that is what it was renamed to

**Review by tikisailor — COMMENTED** [review]
Looks good, just some minor comments

File: src/app/app.ts (hunk 1 above)
Comment (supreme2580) [reply src/app/app.ts:68]: I will definitely check that out and get back to you

File: src/app/events/RateLimit.ts (hunk 2 above)
Comment (supreme2580) [reply src/app/events/RateLimit.ts:17]: I don't think url exists but I will check it out

File: src/app/app.ts (hunk 1 above)
Comment (LordRanchoatos) [reply src/app/app.ts]: yes this is the format for the intents in v14.  GatewayIntentBits.Guilds, GatewayIntentBits.GuildMembers etc.

File: src/app/events/RateLimit.ts (hunk 2 above)
Comment (LordRanchoatos) [reply src/app/events/RateLimit.ts:17]: url do exist tiki is right. `path: rateLimitData.url`
you can checkout the bot template which has been converted to v14. [here](https://github.com/LordRanchoatos/Bot-Template)

Comment (supreme2580) [reply src/app/events/RateLimit.ts:17]: But when I write rateLimitData. url doesn't show up instead path does, everyone seems to be working with different code e.g mine is discord v14.7.0 the bot template is v14.0.3, I modified all necessary for the code to run, when it was on .url yarn start failed on my pc saying that property doesn't exist

Comment (tikisailor) [reply src/app/events/RateLimit.ts:17]: There must be something else wrong. Maybe try to delete the `dist` and `node_modules` folders to make a clean install.  In vscode if you shift + click on the `RateLimitData` type it should show you the interface. Also the event was renamed to `rateLimited`.

**Comment by supreme2580** [comment]
Okay tiki let me try it

**Comment by nonsensetwice** [comment]
Closing this PR to address merge conflicts in new local branch. Changes made directly to this PR were being rejected despite all branches being up-to-date. See branch [`codebase-update-discordjs`](https://github.com/BanklessDAO/mad-hatter/tree/feature/codebase-update-discordjs) for details.

**Pull request closed without merging**